templates/gen_component/lib/components/component.ex.eex

```diff
@@ -0,0 +1,37 @@
+defmodule <%= @app_module_name %>.Component.<%= @component_module_name %> do  
+    require Logger  
+    use Scenic.Component
+    import Scenic.Primitives, only: [{:text, 3}, {:update_opts, 2}]
+
+    @graph Graph.build()
+    |> text("", text_align: :center, translate: {100, 200}, id: :text)
+
+    # TODO: Explanation of info/1 callback here.
```

Comment:
We do not need an extra documentation or comments if this is callback. In that case all we need is proper `@impl` and all the documentation in `behaviour` and optionally guides.

Reply:
Theoretically, yes. Sometimes it's helpful just to give people the reference documentation right where they would be running into it. Phoenix does this a lot in the gen'd code. I'm not super picky either way.

Reply:
Yes, `phoenix` gives many comments, `@doc` and `@moduledoc` for generated modules. The `@impl My.Behaviour` is the definitely minimal we need, so it's enough (right now) just to make easy to understand for maintainers and contributors (otherwise people would spend time on looking for it in code source). Later we can freely focus on extra comments for new developers.

Reply:
Gotcha, went ahead and did that.